docs/application/web/guides/machine-learning/tensors.md

```diff
@@ -0,0 +1,137 @@
+    tensorsInfo.setTensorDimensions(0, [2, 2]);
+    ```
+
+4. To clone and compare `TensorsInfo` object:
+
+    ```javascript
+    var clone = tensorsInfo.clone();
+    var isEqual = tensorsInfo.equals(clone);
+    ```
+
+     > [!NOTE]
+     > Use `equals()` to compare `TensorInfo` objects. Comparisons must not be done with built-in `==`, `===`, `!=`, `!==` operators which may return false results for this data type.
+
+5. To calculate the byte size of tensor data use `getTensorSize`:
+
+    ```javascript
+    var byteSize = tensorsInfo.getTensorSize(0);
+    ```
+
+6. After you create tensor definition, you can use `getTensorsData` to get `TensorsData` object:
+
+    ```javascript
+    var tensorsData = tensorsInfo.getTensorsData();
+    ```
+
+7. Ensure to dispose off the `TensorsInfo` object when you do not need it anymore:
+
+    ```javascript
+    tensorsInfo.dispose();
+    ```
+
+## Manage tensor data
+
+The `TensorsData` object keeps data value of the tensors.
+
+1. To get specific data object from `TensorsData`, use `getTensorRawData`:
+
+    ```javascript
+    var rawData = tensorsData.getTensorRawData(0);
+    ```
+
+    - You can also specify the location and size of data you want to get:
+
+        ```javascript
+        // fetch one element
+        var rawDataOne = tensorsData.getTensorRawData(0, [0, 0], [1, 1]);
+        // fetch first row
+        var rawDataRow = tensorsData.getTensorRawData(0, [0, 0], [-1, 1]);
+        ```
+
+    > [!NOTE]
+    > To gather more information about specifying location and size of the raw data, see `TensorsData.getTensorRawData` (in [mobile](../../api/latest/device_api/mobile/tizen/ml.html#TensorsData::getTensorRawData), [wearable](../../api/latest/device_api/wearable/tizen/ml.html#TensorsData::getTensorRawData), and [TV](../../api/latest/device_api/tv/tizen/ml.html#TensorsData::getTensorRawData) applications).
+
+2. To set data to `TensorsData` object, use `setTensorRawData`:
+
+    ```javascript
+    tensorsData.setTensorRawData(0, [1, 2, 3, 4]);
+    ```
+
+    - You can also specify the location and size of data you want to set:
+
+        ```javascript
+        // set only one element
+        tensorsData.setTensorRawData(0, [7], [0, 0], [1, 1]);
+        // set first row
+        tensorsData.setTensorRawData(0, [4, 5], [0, 0], [-1, 1]);
+        ```
+
+     >  [!NOTE]
+     > To gather more information about specifying location and size of the raw data, see `TensorsData.setTensorRawData` (in [mobile](../../api/latest/device_api/mobile/tizen/ml.html#TensorsData::setTensorRawData), [wearable](../../api/latest/device_api/wearable/tizen/ml.html#TensorsData::setTensorRawData), and [TV](../../api/latest/device_api/tv/tizen/ml.html#TensorsData::setTensorRawData) applications).
+
+3. Ensure to dispose off the `TensorsData` objects when you do not need them anymore:
+
+    ```javascript
+    tensorsData.dispose();
+    ```
+
+## Related information
+
+- Dependencies
```

Comment:
@pwasowski2, @nawab-samsung I missed this bit, could you check why are the dependencies repeated twice here

Reply:
I see, @nawab-samsung has already prepared a fix for this: https://github.com/Samsung/tizen-docs/commit/bcbdae1bf914dab1bc09d529e74801d1d4e2884c

Thanks!